packages/react-cli/src/commands/new.ts

```diff
@@ -76,7 +68,7 @@ export class NewCommand extends Command {
     let tasks = new Listr([
       {
         title: 'Creating react project',
-        task: () => runEmber(argsForEmber),
+        task: async () => await runEmberInteractively(argsForEmber),
```

Comment:
when using this async, shouldn't you await a asynchronous action? It looks like `runEmberInteractively` is returning a synchronous action would you want to return exec?[here](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback)

Reply:
that's actually just to get around a type error. task expects the function to return a promise

Reply:
okay I see that 👍